src/com/tw/go/plugin/git/GitCmdHelper.java

```diff
@@ -33,8 +32,8 @@ public GitCmdHelper(GitConfig gitConfig, File workingDir, ProcessOutputStreamCon
 
     @Override
     public String version() {
-        CommandLine gitLsRemote = Console.createCommand("--version");
-        return runAndGetOutput(gitLsRemote).stdOut().get(0);
+        CommandLine gitCmd = Console.createCommand("--version");
+        return runAndGetOutput(gitCmd, new File("/")).stdOut().get(0);
```

Comment:
Happy to change it to somewhere else that we are sure exists (e.g /tmp, or something OS-agnostic) however the problem I was trying to solve is that at least with the `git-path-material-plugin` the workingDirectory (flyweight dir from GoCD) has not always been created at the time the "use gitCmd or jGit" check is being made so the check was failing, and I didn't feel it was particular important which context `git version` was being run from.

Reply:
Okay. I asked because the earlier change didn't have any working directory specified. My assumption is that it would be the current working directory.